ui/src/org/pentaho/di/ui/spoon/Spoon.java

```diff
@@ -4932,6 +4932,11 @@ public boolean quitFile( boolean canCancel ) throws KettleException {
     }
 
     if ( exit ) {
+      // on windows [...].swt.ole.win32.OleClientSite.OnInPlaceDeactivate can
+      // cause the focus to move to an already disposed tab, resulting in a NPE
+      // so we first move the focus to somewhere else
+      this.selectionLabel.forceFocus();
```

Comment:
Yes thank you. We have a few places where widgets shouldn't be disposed and yet we see weird things sometimes (especially on exit)